src/AnnotationThread.js

```diff
@@ -245,20 +245,11 @@ class AnnotationThread extends EventEmitter {
     }
 
     /**
-     * Fire an event notifying that the comment button has been clicked. Also
-     * show the comment box, and give focus to the text area conatined by it.
+     * Does nothing by default
      *
-     * @param {Event} event - The DOM event coming from interacting with the element.
      * @return {void}
      */
-    onCommentClick() {
-        if (!this.threadNumber) {
-            this.saveAnnotation(TYPES.highlight);
```

Comment:
This logic seems to be missing in the new changeset. Do we no longer need to save the annotation?

Reply:
The only time this will get called is on already saved highlights so the plain highlight would already exist at this point.